streams/src/test/java/org/apache/kafka/streams/processor/internals/RecordCollectorTest.java

```diff
@@ -742,7 +825,7 @@ public void shouldThrowIfTopicIsUnknownWithContinueExceptionHandler() {
             }
         );
 
-        collector.send("topic1", "3", "0", null, null, stringSerializer, stringSerializer, streamPartitioner);
+        assertThrows(StreamsException.class, () -> collector.send(topic, "3", "0", null, null, stringSerializer, stringSerializer, streamPartitioner));
```

Comment:
prop: be more specific of the thrown cause

Reply:
Ack